src/assets/scripts/client/index.js

```diff
@@ -13,6 +14,10 @@ const getInitialAirport = () => {
 
     if (_has(localStorage, STORAGE_KEY.ATC_LAST_AIRPORT)) {
         airportName = _lowerCase(localStorage[STORAGE_KEY.ATC_LAST_AIRPORT]);
+
+        if (!AirportController.hasAirport(airportName)) {
```

Comment:
We can simplify this logic by combining the conditions of these two if statements. We are initially setting to the default airport, and then _if an airport is in local storage and it is accessible, THEN we look for that airport_.